<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseFormat</class>
 <widget class="QDialog" name="ChooseFormat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose format</string>
  </property>
  <widget class="QRadioButton" name="radioButton2000">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>CoNLL 2000</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_MalTab">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>9</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>MalTab</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2002">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>37</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>CoNLL 2002</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_gale">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>37</y>
     <width>189</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Gale Alignment Format</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2003">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>65</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>CoNLL 2003</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_giza">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>65</y>
     <width>188</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Giza Alignment Format</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2004">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>93</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>CoNLL 2004</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_the_beast">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>93</y>
     <width>152</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>The Beast Format</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2005">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>121</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>CoNLL 2005</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_lisp_s_expr">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>121</y>
     <width>158</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Lisp S-expr Format</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2006">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>149</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>CoNLL 2006</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_bionlp_2009">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>149</y>
     <width>252</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>BioNLP2009 Shared Task Format</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2008">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>177</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>CoNLL 2008</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2009">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>205</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>CoNLL 2009</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>233</y>
     <width>176</width>
     <height>27</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChooseFormat</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChooseFormat</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
